docker/runtime/python-2.7/kubeless.py

```diff
@@ -1,31 +1,24 @@
 #!/usr/bin/env python
 
-import sys
 import os
 import imp
 
-from bottle import route, run, request
+import bottle
```

Comment:
It all gets loaded anyway.  This is just the "more normal" convention for python because it makes mocking during unittests a lot easier - not that there are any such tests for this code at present.